src/javascript/binary/pages/trade/event.js

```diff
@@ -266,5 +270,6 @@ var highBarrierElement = document.getElementById('barrier_high');
 if (highBarrierElement) {
     highBarrierElement.addEventListener('input', debounce( function (e) {
         processPriceRequest();
+        submitForm(document.getElementById('websocket_form'));
```

Comment:
These lines of code are repeating themselves (the whole ifs).
We should have them in a single function called few times.